nautobot_floor_plan/forms.py

```diff
@@ -41,18 +41,7 @@ class Meta:
         """Meta attributes."""
 
         model = models.FloorPlan
-        fields = [
-            "location",
-            "x_size",
-            "y_size",
-            "tile_width",
-            "tile_depth",
-            "x_axis_labels",
-            "x_origin_seed",
-            "y_axis_labels",
-            "y_origin_seed",
-            "tags",
-        ]
+        fields = "__all__"
```

Comment:
Does this (and the similar change below) have any impact on the form presentation?

Reply:
Yes on the first form, no on the second. I will add field_order in.